unity-renderer/Assets/ABConverter/AssetBundleMetadata/AssetBundleMetadata.cs

```diff
@@ -0,0 +1,8 @@
+
+[System.Serializable]
+public class AssetBundleMetadata
+{
+    public long timestamp = -1;
+    public float version = 1.0f;
```

Comment:
I think having a `string` is better here so we can have meaningful versioning i.e. `1.001.120`.